voucher/templates/enroll.html

```diff
@@ -13,6 +13,7 @@ <h2>Confirm Course</h2>
             <form method="post" enctype="multipart/form-data" class="voucher-form ">
                 {% csrf_token %}
                 <select name="coupon_version" id="coupon_version" classname="form-control">
+                    <option value="">Choose Course</option>
```

Comment:
What happens if the user tries to submit the form when this option is still chosen?

Reply:
Currently its giving error. I will address it too in my PR.